Task 3/moneyexchange/moneychange.md

```diff
@@ -0,0 +1,8 @@
+problem statement - You have coins of 5 and 10 rupees and a particular amount has to be paid. find the minimum number of coins required to pay the amount.
+note: the amount is always payable through the coins of 5 and 10 only.
+input: single line containing an integer x (the amount to be paid)
+ouput: minimum number of coins required to pay the amount
```

Comment:
You can use markdown formatting to make this look better!

Something like this:

```md
#### Input
```
Displays as 

#### Input

in markdown (.md)